src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -147,8 +148,8 @@
             <argument type="service" id="Psr\Container\ContainerInterface" />
         </service>
 
-        <service id="translator_listener" class="Symfony\Component\HttpKernel\EventListener\TranslatorListener">
-            <argument type="service" id="translator" />
+        <service id="locale_aware_listener" class="Symfony\Component\HttpKernel\EventListener\LocaleAwareListener">
```

Comment:
this service should not be defined in `translation.xml`, as it may be useful even when the translation component is not used in the project (as there might be other local-aware services).